src/pages/Risk.tsx

```diff
@@ -12,7 +12,7 @@ function Risk(): ReactElement {
     <div className="riskpage">
       <TopMenu activeTab={t("risk")} />
       <div className="content">
-        <h1 className="title">Risk</h1>
```

Comment:
In the future please try to keep unrelated changes like this to their own PRs - 26 files in one PR is a lot to review 😅

Reply:
Yep 😊